applications/luci-app-statistics/luasrc/controller/luci_statistics/luci_statistics.lua

```diff
@@ -36,7 +36,7 @@ function index()
 		local plugin = dofile(plugin_dir .. filename)
 		local name = filename:gsub("%.lua", "")
 		table.insert(collectd_menu[plugin.category], name)
-		labels[name] = plugin.label
+		labels[name] = _(plugin.label)
```

Comment:
That will not work. The `_()` function must be invoked with a literal string since it is parsed by a static source code scanner to extract translatable phrases.

Reply:
I see… So how should this be done? Should it be invoked in each individual .lua file in the plugins directory?

Reply:
Right, in each individual file, you'll need something like  `label = translate("Foo")`, then it should work. But I didn't look in detail

Reply:
Oh, why translate("Foo") and not _("Foo") like in the original file? Is the translate function globally visible?

Reply:
OK, simply changing "Foo" to translate("Foo") doesn't work:
```
/usr/lib/lua/luci/statistics/plugins/apcups.lua:7: attempt to call global 'translate' (a nil value)
```
Any hints for how to make this work?

Reply:
I have done the following to add the translation. See https://github.com/openwrt/luci/blob/master/applications/luci-app-statistics/luasrc/statistics/rrdtool/definitions/apcups.lua#L7
please try if this helps.

Reply:
Hey @feckert, I think I got this working correctly. I'm now loading the file with loadfile rather than dofile, and I use `setfenv(f, { _ = luci.i18n.translate })` to give the loaded code access to the i18n stuff. The source code scanner that @jow- mentioned should probably work like before because the code looks the same `_("Bla")`.